setup.py

```diff
@@ -210,13 +205,9 @@ class BuildPythonProtosCommand(Command):
     ]
 
     def initialize_options(self):
-        self.python_protoc = [
-            sys.executable,
-            "-m",
-            "grpc_tools.protoc",
-        ]  # find_executable("protoc")
+        self.buf = [find_executable("buf")]
```

Comment:
How is `buf` dependency pulled in? Does it have to be `brew install`ed or somthing?

Reply:
I has a `brew install` or can `go install` it

https://docs.buf.build/installation